Jenkinsfile.cloud

```diff
@@ -86,7 +86,7 @@ node(env.NODE) {
             rpm-ostree db diff --repo=${repo} \
                 ${prev_commit} ${commit} > ${dirpath}/${commit}.pkgdiff
         """
-        sh "sha256sum ${qcow}.gz | awk "{print $1}" > ${qcow}.gz.sha256sum"
+        sh "sha256sum ${qcow}.gz | awk '{print $1}' > ${qcow}.gz.sha256sum"
```

Comment:
I spent a while reading http://groovy-lang.org/syntax.html#_string_interpolation while working on this project.   Anyways, this looks a lot more right, except based on my testing, we need `\$1` instead of `$1` to tell Groovy not to look for a variable named `1`.  The single quotes ensure `/bin/sh` won't try to look it up, because we want the *third* language involved here (awk) to find the `1` variable.  Phew!

Reply:
![groooooooooovy](https://user-images.githubusercontent.com/10236257/40329689-5a63bc62-5d18-11e8-9e9d-7ed0635c649c.jpg)